centreon/www/include/Administration/myAccount/formMyAccount.php

```diff
@@ -106,14 +106,18 @@
 $form = new HTML_QuickFormCustom('Form', 'post', "?p=" . $p);
 $form->addElement('header', 'title', _("Change my settings"));
 $form->addElement('header', 'information', _("General Information"));
-$form->addElement('text', 'contact_name', _("Name"), $attrsText);
-if ($cct["contact_auth_type"] != 'ldap') {
+if ($cct["contact_auth_type"] === 'local') {
```

Comment:
Is this working for user connecting through openid ?

Reply:
its working only for user with contact_auth_type = local for otheir using ldap or SSO the alias, Name, Email, Password Management theyre not allowed to it